Libraries/WebSocket/RCTSRWebSocket.m

```diff
@@ -245,7 +245,7 @@ + (void)initialize;
   CRLFCRLF = [[NSData alloc] initWithBytes:"\r\n\r\n" length:4];
 }
 
-- (instancetype)initWithURLRequest:(NSURLRequest *)request protocols:(NSArray<NSString *> *)protocols options:(NSDictionary<NSString *, NSString *> *)options
+- (instancetype)initWithURLRequest:(NSURLRequest *)request protocols:(NSArray<NSString *> *)protocols headers:(NSDictionary<NSString *, NSString *> *)headers
```

Comment:
Since NSURLRequest already includes headers, is there any reason for the seperate headers parameter? Can't we just include them in the request?